File: src/test/java/aoc2015/day02/Day02Test.java
package aoc2015.day02;

import static org.junit.jupiter.api.Assertions.assertEquals;

import org.junit.jupiter.api.Test;

class Day02Test {

  @Test
  void part1Test() {
    var day = new Day02();

    assertEquals(58, day.part1("2x3x4"));
    assertEquals(43, day.part1("1x1x10"));
  }

  @Test
  void part2Test() {
    var day = new Day02();

    assertEquals(34, day.part2("2x3x4"));
    assertEquals(14, day.part2("1x1x10"));
  }
}

File: src/test/java/aoc2020/day02/Day02Test.kt
package aoc2020.day02

import aoc.loadExample
import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

class Day02Test {

    val input: String = loadExample(2020, "day02.txt")

    @Test
    fun part1Example() {
        val subject = Day02()

        assertThat(subject.part1(input))
            .isEqualTo(2)
    }

    @Test
    fun part2Example() {
        val subject = Day02()

        assertThat(subject.part2(input))
            .isEqualTo(1)
    }
}